src/libraries/System.Private.CoreLib/src/System/Runtime/Versioning/PlatformAttributes.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.Versioning
```

Comment:
@ViktorHofer @safern I am getting a really strange compilation error for **live-builds** only:

```log
src/libraries/System.Private.CoreLib/src/System/Runtime/Versioning/PlatformAttributes.cs(4,0): error : (NETCORE_ENGINEERING_TELEMETRY=Build) syntax error at token 'namespace' in: namespace System.Runtime.Versioning
```

do you have any idea what can be going on here? this **should** compile. I can't repro it locally and it seems to be specific to **live-builds** only.

Reply:
It seems like the attribute is included in an `.ilproj` that is why it is failing. I think we should not include this attribute in an `.ilproj` as it is not `IL` that's why you're getting the error.

Reply:
@safern big thanks for help! it just proves that including this file in all projects was a bad idea ;)